### PR TITLE
Set up Vue page

### PR DESCRIPTION
**Pull request by hcorson-dosch**
This PR:
* Sets up the basic Vue repo (based on Vue scratch)
* Adds a component with a zoomable image, using the Vue component [`vue-zoomer`](https://github.com/jarvisniu/vue-zoomer)
  * This component allows scroll-wheel zoom on desktop - zooms in on mouse location
  * Button zoom on desktop and mobile/tablet
  * Should also allow touch zoom on mobile/tablet -- need to push to beta to test
* Sets up a pipeline to push png and webp images in two languages to S3
* Adds buttons for zoom navigation
* Adds button for switching language of image
* Trys to set up flex layout so that image container always fills remaining space (after header, footer, and button container) to get to a full page height
  * For image, flex settings seem to be conflicting with some of the zoom component settings, so still have to set container height based on viewport, w/ different setting for different window sizes... not ideal

### CODE REVIEW

**Comment by ceenell** [comment]
Really good! Left a few comments to clean up the Vue repo and can otherwise be merged. This ran for me locally, and I agree was a bit slow, but we can work on addressing that as we have the capacity in the coming weeks. For now, let's get it up for testing and the release

**Comment by hcorson-dosch** [comment]
> Left a few comments to clean up the Vue repo and can otherwise be merged.

Wait @ceenell  I can't see your comments :(

**Review by ceenell — APPROVED** [review]

File: jenkins/Jenkinsfile.build [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        }
+        stage('checkout'){
+            steps{
+                checkout([$class: 'GitSCM',
+                          branches: [[name: "${params.BRANCH_TAG}"]],
+                          doGenerateSubmoduleConfigurations: false,
+                          extensions: [],
+                          gitTool: 'Default',
+                          submoduleCfg: [],
+                          userRemoteConfigs: [[url: 'https://github.com/usgs-vizlab/gw-conditions.git']]
```
[the remark this answers is not in the document]
Comment (hcorson-dosch) [reply jenkins/Jenkinsfile.build]: good catch!

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      <meta property="og:title" content="">
+      <meta property="og:description" content="">
+      <meta property="og:image" content="">
+      <!-- Twitter -->
+      <meta property="twitter:card" content="summary_large_image">
+      <meta property="twitter:url" content="">
+      <meta property="twitter:title" content="">
+      <meta property="twitter:description" content="">
+      <meta property="twitter:image" content="">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
```
[the remark this answers is not in the document]
Comment (hcorson-dosch) [reply public/index.html:27]: ooh yeah. Is that thumbnails that need to be prepped?

[the remark this answers is not in the document]
Comment (ceenell) [reply public/index.html:27]: yes a square thumbnail that doulbes for the viz portfolio should work.

[the remark this answers is not in the document]
Comment (hcorson-dosch) [reply public/index.html:27]: added an issue: #2

**Pull request merged**